hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -163,6 +169,111 @@ public void renameTable(TableIdentifier from, TableIdentifier to) {
     }
   }
 
+  @Override
+  public boolean createNamespace(Namespace namespace) {
+    Preconditions.checkArgument(!namespace.isEmpty(),
+        "Namespace could not empty. ");
+    Preconditions.checkArgument(namespace.levels().length == 1,
+        "Hive MetaStore cannot support multi part namespace now: %s", namespace.toString());
+    try {
+      clients.run(client -> {
+        client.createDatabase(nameSpaceToHiveDb(namespace));
+        return null;
+      });
+
+      return true;
+
+    } catch (AlreadyExistsException e) {
+      throw new org.apache.iceberg.exceptions.AlreadyExistsException("namespace already exists: %s",
+            namespace.toString());
+    } catch (TException e) {
```

Comment:
Add a newline above this to match style.

Reply:
thanks Ryan , now has fixed it.